src/controllers/index.js

```diff
@@ -0,0 +1,9 @@
+import joinRandomGame from './join-random-game';
```

Comment:
If you're going to do it like this just include the functions in the file.  Annoying to switch between files to see functionality.